app/utils/Strings.scala

```diff
@@ -32,6 +34,8 @@ object Strings {
     Random.alphanumeric take length mkString
   }
 
+  def zeroOrOne = Gen.oneOf(0, 1).sample.getOrElse(0).toString
```

Comment:
same here.

Reply:
I can see the value in adding a `numberBetween` method, but at the same time, this matches what we have, and I don't see there being a use for anything beyond "1 to 9", "0 to 9" and "0 or 1". 

I'm okay with leaving this from a readability point of view.